ls8/cpu.py

```diff
@@ -7,6 +7,9 @@ class CPU:
 
     def __init__(self):
         """Construct a new CPU."""
+        self.ram = [0] *256
```

Comment:
why are you using  [0] *256 while using [0] * 8 in the next line?

Reply:
I thought the ram and registers were two separate things? So [0] * 8 is for the registers